test/unit/src/audio/AudioContext.tests.js

```diff
@@ -4,29 +4,41 @@ import { AudioContext } from '../../../../src/audio/AudioContext.js';
 
 export default QUnit.module( 'Audios', () => {
 
-	QUnit.module( 'AudioContext', () => {
+	QUnit.module( 'AudioContext', ( hooks ) => {
 
 		function mockWindowAudioContext() {
 
-			if ( typeof window === 'undefined' ) {
+			global.window = {
+				AudioContext: function () {
 
-				global.window = {
-					AudioContext: function () {
+					return {
+						createGain: () => {
 
-						return {
-							createGain: () => {
+							return {
+								connect: () => {},
+							};
 
-								return {
-									connect: () => {},
-								};
+						}
+					};
 
-							}
-						};
+				},
+			};
 
-					},
-				};
+		}
+
+		if ( typeof window === 'undefined' ) {
+
+			hooks.before( function () {
+
+				mockWindowAudioContext();
```

Comment:
Shouldn't you then remove explicit calls to this function later, e.g. on line 48?

Reply:
You mean where that `after` hook queues to clean up from this function 2 lines below ?